src/brogue/IO.c

```diff
@@ -3474,7 +3479,7 @@ void flavorMessage(char *msg) {
 // arrived on the same turn, they may collapse.  Alternately, they may collapse
 // if the older message is the latest one in the archive and the new one is not
 // semi-colon foldable (such as a combat message.)
-void message(const char *msg, enum messageFlags flags) {
+void message(const char *msg, enum messageFlags flags, short speechPriority) {
```

Comment:
Any reason to not fold `speechPriority` into the other `messageFlags`?  With a judicious default it could probably save touching a lot of `message` call sites.

Reply:
a great idea, i wanted to avoid modifying the existing enum, but that handles a default much better